src/predict.rs

```diff
@@ -473,8 +476,38 @@ pub(crate) mod native {
         height,
         bit_depth,
       ),
-      PredictionMode::H_PRED => pred_h(dst, left_slice, width, height),
-      PredictionMode::V_PRED => pred_v(dst, above_slice, width, height),
+      PredictionMode::H_PRED => {
```

Comment:
Alternatively, specialise here and add `H_PRED` and `V_PRED` at line 523.
```suggestion
      PredictionMode::H_PRED if angle == 180 => {
```